lib/core/config/locales/en.yml

```diff
@@ -3,3 +3,10 @@ en:
     errors:
       messages:
         json_schema_mismatch: "does not match specified schema: %{errors_summary}"
+  activemodel:
```

Comment:
Is this really needed? seems to me that at most, we would need to rename line 2 to activemodel if rails is not picking up the error from activerecord

Reply:
Furthermore, you're adding an error message specific to your fake model in the specs, so this definitely should not live here